src/screens/ExplainSheet.js

```diff
@@ -36,6 +36,7 @@ import {
 } from '@/utils';
```

Comment:
This whole file is WAY TOO LONG
We should try splitting it into smaller pieces

Reply:
yep